### ISSUE TITLE
Inconsistent regularization loss computation when used with pretrained models. 

### ISSUE THREAD

**Issue by meet-minimalist** [tensorflow/tensorflow#41161]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): **Yes**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): **Google Colab Environment**
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: **N/A**
- TensorFlow installed from (source or binary): **Binary (Colab Pre-installed)**
- TensorFlow version (use command below): **2.4.0-nightly**
- Python version: **3.6.9**
- Bazel version (if compiling from source): **N/A**
- GCC/Compiler version (if compiling from source): **N/A**
- CUDA/cuDNN version: **N/A (Colab CPU Environment used)**
- GPU model and memory: **N/A**

**Describe the current behavior**
I am trying to use MobileNetV2 pretrained model for a simple classification task. Here I want to add regularization loss for MobileNetV2 - base model along with added FC layers. I have conducted 3 experiments as per attached colab notebook, where I get different results for the computation of regularization loss. The MobileNetV2 regularizer loss is computed twice in the model. And I dont know why it happens.

**Describe the expected behavior**
The computation of the loss in all the experiments should be same.

**Standalone code to reproduce the issue**
Colab Link : https://colab.research.google.com/drive/1CeKMIAq_g0AOKdakupKIeUjEhkL_TtI1?usp=sharing

**Other info / logs** 
1. I have tried the same on TF2.2 but got the same behaviour.
2. I have got the reference of this: [issue-37511](https://github.com/tensorflow/tensorflow/issues/37511#issuecomment-599173876 ) for adding regularization loss into pretrained model.

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with [TF v2.2](https://colab.research.google.com/gist/amahendrakar/b930aa726bbb49453829e891f0dad42b/41161-2-2.ipynb) and [TF-nightly](https://colab.research.google.com/gist/amahendrakar/82ab859e8adb6a7eb87f70e49c840e01/41161.ipynb). Please find the attached gist .Thanks!

**Comment by saikumarchalla** [comment]
Was able to reproduce  the issue with TF 2.5 and Nightly versions.Please find the gist [here](https://colab.research.google.com/gist/saikumarchalla/5ed96954861babbb70def2f6ac5fc7e8/41161-2-2.ipynb#scrollTo=JfUG_vigEQLl).Thanks!

**Comment by Venkat6871** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
